demoapp/demo_app_functions.py

```diff
@@ -182,9 +211,37 @@ def get_chap_sec_names(df: pd.DataFrame, bill_number: str, mgl_names_file_path:
             except Exception as e:
                 continue
         mgl_names = ", ".join([f"{key}: {value}" for dct in names for key, value in dct.items()])
-       
+
         return mgl_names
+
+def get_chap_sec_names_internal(chap_sec_lists: list, mgl_names_file_path: str = "./chapter_section_names.pq") -> str:
+    """
+    Fetches chapter and section names for a given bill number from a DataFrame.
+    
+    Args:
+        chap_sec_lists (list): list of tuples containing chapter number and section numbers.
+        mgl_names_file_path (str): path for the file containing chapter and section names
```

Comment:
Do something like this.

```suggestion
        mgl_names_file_path (str): path for the parquet file containing chapter and section names. Expected columns are `Chapter_Number` and `Section_Number`.
```

Is there another column expected for the `return` line, like a 'Chapter_Name' column or similar?